src/Application.vala

```diff
@@ -97,6 +96,7 @@ public class Application : Granite.Application {
             command_line.connect (on_command_line);
         }
 
+        Gtk.Settings.get_default().set("gtk-application-prefer-dark-theme", true);
```

Comment:
perhaps we should use
`Gtk.Settings.get_default ().gtk_application_prefer_dark_theme = true`

Reply:
Yep, probably! I was just moving the existing line, but we usually prefer setting the property directly instead of using the setter.